source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1545,14 +1552,9 @@ shutting down" errors:
   * - ShutdownInProgress
     - 91
 
-If the client is connected to server version 4.2 or higher, and the client
-receives a "not master" or "node is recovering" error which is not a
-"node is shutting down" error, the client MUST keep any connections it has to
-the server open, and MUST NOT clear its connection pool for the server.
-If the client is connected to server version 4.2 or higher and receives a
-"node is shutting down" error, or if the client is connected to server version
-4.0 or lower and receives a "not master" or "node is recovering" error,
-the client MUST clear its connection pool to the server.
+When handling a "not master" or "node is recovering" error, the client
+clears the server's connection pool if and only if the error is
```

Comment:
Since this is a requirement, I am under the impression that the current preference is to express it as "MUST clear" instead of "clears".

Reply:
Done.